Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -810,6 +810,37 @@ namespace PlayerCommands
 			base->Save();
 			PrintUserCmdText(client, L"OK Module destroyed");
 		}
+		else if (cmd == L"switch")

```

Comment:
switch is a weird command name for this

Reply:
remade as "pause" and "resume" separate commands